### PR TITLE
Display some missing shortcuts in Browse form menus

### PR DESCRIPTION
**Pull request by pmiossec**
* create tag (ctrl+ t)
* create branch (ctrl + b)
* close repository (ctrl + w)

### CODE REVIEW

**Comment by RussKie** [comment]
The description has it inverted - ctrl+b creates new branch, ctrl+t - tag.
Please fix the commit message.

I can't see the diff on my phone.

On 08/01/2018 5:59 AM, "Philippe Miossec" <notifications@github.com> wrote:


   - create tag (ctrl+b)
   - create branch (ctrl +t)
   - close repository (ctrl +w)

------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/gitextensions/gitextensions/pull/4330
Commit Summary

   - Display some missing shortcuts

File Changes

   - *M* GitUI/CommandsDialogs/FormBrowse.Designer.cs
   <https://github.com/gitextensions/gitextensions/pull/4330/files#diff-0>
   (3)

Patch Links:

   - https://github.com/gitextensions/gitextensions/pull/4330.patch
   - https://github.com/gitextensions/gitextensions/pull/4330.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/gitextensions/gitextensions/pull/4330>, or mute the
thread

**Comment by pmiossec** [comment]
That was not in the commit message but just in the PR text  (and I already fixed it. Your mail is deprecated ;))

**Comment by pmiossec** [comment]
>I can't see the diff on my phone. 

Viva el `design.cs` files ;)

File: GitUI/CommandsDialogs/FormBrowse.Designer.cs [hunk 1]
```diff
@@ -800,6 +800,7 @@ private void InitializeComponent()
             // 
             this.closeToolStripMenuItem.Name = "closeToolStripMenuItem";
             this.closeToolStripMenuItem.Size = new System.Drawing.Size(221, 22);
+            this.closeToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Control | System.Windows.Forms.Keys.W)));
```
Comment (RussKie) [inline GitUI/CommandsDialogs/FormBrowse.Designer.cs]: Will it change if changed by the user in Settings?

Comment (pmiossec) [reply GitUI/CommandsDialogs/FormBrowse.Designer.cs]: No. But that are not actions where you could changes there shortcuts (and these shortcuts are already displayed in the context menu...)

Comment (RussKie) [reply GitUI/CommandsDialogs/FormBrowse.Designer.cs]: This one is actually configurable, that prompted my original question:

image

**Comment by pmiossec** [comment]
Oups! I verified the 2 others...

**Comment by pmiossec** [comment]
done! With some others missing...

**Pull request merged**